go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/p4tin/goaws
+module github.com/Admiral-Piett/goaws
 
 go 1.12
```

Comment:
Probably need to update this as well

Reply:
Yep, I was going to do that shortly.  Want to use this to get us back in working order, then start upgrading.